halibot/message.py

```diff
@@ -26,12 +26,26 @@ def makeOrigin(self):
 
 class Message():
 
+	class Author(str):
+		def __init__(self, a, id=""):
+			self.id = id
+			self.name = str(a) if a else ""
+		def __str__(self):
+			return self.name
+		def __repr__(self):
+			return self.name
+		def __setattr__(self, key, value):
+			if key in ("id", "name"):
+				object.__setattr__(self, key, value)
+		def __getattr__(self, key):
+			return str(object.__getattr__(self,key))
```

Comment:
I don't think `__getattr__` is needed... If you want to force everything to a string, you can cast `value` to a string in `__setattr__`.